guides-and-standards/guide-code-review.md

```diff
@@ -0,0 +1,144 @@
+
+### How should I phrase my feedback?
+
+Feedback should always focus on the code itself and not on the author. Avoid using language that is accusatory, judgmental, or personal. Instead, use language that is objective, specific, and constructive.
+
+Here are some examples:
+
+- Avoid saying "you should have done X" or "this code should do X"; instead, say "I feel that this code would be clearer if it did X".
+- Avoid saying "you made a mistake here" or "there is a mistake here"; instead, say "It looks to me like there may be a mistake here because of X".
+- Avoid saying "this is confusing"; instead, say "This may be confusing because of X. It would be clearer if it were Y."
+
+### How should I prioritize my feedback?
+
+Before making a comment, first think carefully about whether the comment warrants the author's attention. In particular, think about whether your comment is likely to improve the code in a meaningful way _for other developers_ or whether it is instead more a reflection of personal preference. For example, comments about variable names are often subjective (as long as the name is descriptive), but comments about _inconsistent_ variable names are usually worth making, as they are likely to make the code hard to understand for others.
+
+#### What is a "nit"?
+
+For comments that you feel are worth making but that you wish to indicate are either minor or may be subjective, the prefix "nit" (short for "nitpick") is often used. For example, "Nit: capitalize this figure label" or "Nit: use `dataframe.empty()` instead of `len(dataframe) == 0`". Usually, it is understood that the author may choose to accept or reject such comments at their discretion, without further discussion.
+
+### When should I approve a PR?
+
+You'll likely notice at least a few important issues during your initial review that must be addressed before the PR can be merged (note that these could be "minor but major," like typos in variable names). In these cases, you should submit your initial review with a "changes requested" status. Once the author has adequately addressed your feedback, you'll need to re-review the PR and submit a second, approving review. Usually, this step is a formality and does not involve additional comments or changes.
+
+If, on the other hand, your initial review does not surface any critical issues, then it is acceptable to skip the re-review process and submit your initial review as an approval (sometimes called "approving with comments"). In this case, there is usually an implicit understanding that any comments in your review are minor enough that the author can address them at their descretion.
+
```

Comment:
consider adding a note on turnaround time for PRs: once someone sent code for your review, you're effectively blocking them from moving forward, so it is important to review the PR as soon as possible (while not preempting important tasks you're working on such as your own code as an author). If you're unlikely to respond within the "2 business day" timeframe discussed earlier, reach out to the author to explain this.

Reply:
Added this in a new section "how long do I have to respond to a review request?"